website/src/components/LeaderboardTable/LeaderboardTable.tsx

```diff
@@ -61,14 +62,34 @@ export const LeaderboardTable = ({
       },
       columnHelper.accessor("display_name", {
         header: t("user"),
-        cell: ({ getValue, row }) =>
-          isAdminOrMod ? (
-            <Link as={NextLink} href={`/admin/manage_user/${row.original.user_id}`}>
-              {getValue()}
-            </Link>
-          ) : (
-            getValue()
-          ),
+        cell: ({ getValue, row }) => (
+          <div className="flex flex-row items-center gap-2">
+            {row.original.image ? (
+              <Image
+                src={`${row.original.image}`}
+                alt={`${getValue()}'s avatar`}
+                width={30}
+                height={30}
+                className="rounded-full"
+              />
+            ) : (
+              <Image
+                src={`https://api.dicebear.com/5.x/initials/png?seed=${getValue()}&radius=50&backgroundType=gradientLinear`}
```

Comment:
I noticed that you are calling `getValue` more than once, maybe should we cache it?  I realise this prevents the explicit return.  Additionally, might it make sense to create a component for this i.e `<UserAvatar displayName={displayname} />` incase we want to use it again?